components/bookmark-control.jsx

```diff
@@ -100,15 +109,7 @@ class BookmarkControl extends React.Component {
     ReactGA.event(this.createGaEventConfig(this.state.bookmarked ? `Remove fav` : `Add fav`));
 
     if (user.loggedin) {
-      this.toggleBookmark((error) => {
-        if (error) return;
-
-        this.setState({ bookmarked: !this.state.bookmarked }, () => {
-          this.props.updateCardBookmarkedState(this.state.bookmarked);
-        });
-      });
-
-      return;
+      this.toggleBookmark();
```

Comment:
this change made `callback` `undefined`

<img width="851" alt="screen shot 2017-12-06 at 2 13 06 pm" src="https://user-images.githubusercontent.com/2896608/33688280-b920fdf2-da8f-11e7-89cf-a4e50a35ff2c.png">